src/Raven.Server/Utils/BackupUtils.cs

```diff
@@ -231,7 +231,8 @@ private static List<string> AddDestinations(PeriodicBackupStatus backupStatus)
 
     public static NextBackup GetNextBackupDetails(NextBackupDetailsParameters parameters)
     {
-        var nowUtc = SystemTime.UtcNow;
+        var nowUtc = DateTime.UtcNow;
```

Comment:
Why do we want to replace it by `DateTime`? `SystemTime` is something that we can control even in tests. If it's not changed then it should be the same thing effectively

Reply:
I didn't use SystemTime because of the comment
https://github.com/ravendb/ravendb/blob/a12d9be6c5f0fcdef3d09b3025f00f8770e0c642/src/Raven.Client/Util/SystemTime.cs#L16
And I don't have DocumentDatabase there.
Also I want it to be consistent with other places in backup that we mostly use DateTime